docs/dynamic-component-loading.md

```diff
@@ -75,12 +75,12 @@ The second overload adds the ability to specify:
 
 ## Handling of various asset types
 What happens with various asset types once the ALC decides to load it in isolation.
-* Normal managed assembly (code) - The `.deps.json` parsing code will return list of full file paths. The ALC will just find it there and load it.
+* Normal managed assembly (code) - The `.deps.json` parsing code will return list of full file paths. The ALC will just find it there and load it.  
+Note that R2R images are handled by this as well since they are basically just a slightly different managed assembly.
```

Comment:
We should mention that if the same R2R image is loaded by two ALCs, only one of them can use the native code. This is not a limitation of the plugin model per say, but we should mention it in the context.